packages/react/src/components/Autofill/Autofill.test.tsx

```diff
@@ -291,4 +291,34 @@ describe('Autofill', () => {
       ['🆘', false],
     ]);
   });
+
+  it('does not allow changes to the input value when (defaultVisibleValue) prop is provided', () => {
```

Comment:
Generally a component *should* allow changes when a `defaultValue`-like prop is provided. Is Autofill different?

Reply:
Good catch! I don't believe the tests were running properly and never captured the input update.  This should have been `does allow changes`.